iOS/Flows/Conversation/Threads/ThreadViewController.swift

```diff
@@ -164,11 +179,10 @@ extension ThreadViewController: TransitionableViewController {
 }
 
 // MARK: - Updates and Subscription
+
 extension ThreadViewController {
 
     func subscribeToUpdates() {
-        self.addKeyboardObservers()
```

Comment:
Auto keyboard adjustments mess up the time machine layout and aren't needed anyway.